conf/services/conflationTypes.json

```diff
@@ -25,12 +25,12 @@
         "matcher": "",
         "merger": "",
         "members": {
-            "differential.remove.unconflatable.data": "Pass unconflatable data from the secondary input to output",
+            "differential.remove.unconflatable.data": "Remove unconflatable data in the secondary input from the output",
+            "differential.treat.reviews.as.matches": "Treat reviews as matches and remove from the output",
             "differential.snap.unconnected.roads": "Snap unconnected secondary roads to reference roads",
-	    "differential.treat.reviews.as.matches": "Treat reviews as matches and remove from output",
-            "snap.unconnected.ways.snap.tolerance": "Maximum distance, in meters, to allow snapping unconnected roads to neighboring roads",
-            "snap.unconnected.ways.use.existing.way.nodes": "Reuse highway nodes when snapping unconnected roads",
-            "snap.unconnected.ways.existing.way.node.tolerance": "Maximum distance, in meters, to allow snapping unconnected highway nodes to neighboring roads"
+            "snap.unconnected.ways.snap.tolerance": "Maximum distance, in meters, to allow snapping unconnected roads to existing roads",
+            "snap.unconnected.ways.use.existing.way.nodes": "Reuse existing road nodes when snapping unconnected roads",
+            "snap.unconnected.ways.existing.way.node.tolerance": "Maximum distance, in meters, to allow snapping unconnected roads to existing road nodes"
```

Comment:
Renamed some descriptions to be more clear.
Reordered so list of options makes sense in the UI (some checkboxes relate to subsequent options)